### PR TITLE
Added Catalan translations

### PR DESCRIPTION
**Pull request by ArnyminerZ**
Added the Catalan language into the `translations` directory.
Also added the localization label for all PRs related to translations.

### CODE REVIEW

**Comment by ArnyminerZ** [comment]
Still missing services localization, and maybe sensors, but still finding out how to do it.

File: .github/labels.yml [hunk 1]
```diff
@@ -73,6 +73,10 @@
   description: Removals and Deprecations
   color: 9ae7ea
 
+- name: localization
```
Comment (KapJI) [inline .github/labels.yml:76]: Then you need to assign some section in release drafter for it. Probably just create the new one. Worth splitting as a separate PR.

File: custom_components/google_home/types.py [hunk 2]
```diff
@@ -70,11 +72,19 @@ class AlarmsAttributes(TypedDict):
 class TimersAttributes(TypedDict):
     """Typed dict for timers attributes"""
 
+    device_class: str
     next_timer_status: str
     timers: list[GoogleHomeTimerDict]
     integration: str
 
 
+class DoNotDisturbAttributes(TypedDict):
```
Comment (KapJI) [inline custom_components/google_home/types.py:81]: This doesn't look related to translation, please submit a separate PR for it.

File: custom_components/google_home/const.py [hunk 3]
```diff
@@ -84,6 +84,11 @@
 JSON_TIMER: Final[str] = "timer"
 JSON_NOTIFICATIONS_ENABLED: Final[str] = "notifications_enabled"
 
+DEVICE_CLASS_DEVICE: Final[str] = "google_home__device"
+DEVICE_CLASS_ALARMS: Final[str] = "google_home__alarms"
+DEVICE_CLASS_TIMERS: Final[str] = "google_home__timers"
+DEVICE_CLASS_DO_NOT_DISTURB: Final[str] = "google_home__do_not_disturb"
+
```
Comment (KapJI) [inline custom_components/google_home/const.py:91]: These are not valid values. List of supported device classes for `sensor` platform: https://www.home-assistant.io/integrations/sensor/

**Review by KapJI — CHANGES_REQUESTED** [review]

**Review by leikoilja — COMMENTED** [review]
Great initiative to implement translations, thank you 🙌  
Once it's merged i ll add Russian language as well 🚀 
Let's split the proposed changes into 3 separate PRs and clean up a little, this is really important because it makes it much easier to review and keeps git history much better structured and organised. 
We can have:
- 1 PR for adding new labeler and draft releaser section
- 1 PR for Catalan translations
- 1 PR for refactoring (like adding new classes and attributes)

**Review by KapJI — APPROVED** [review]
Thanks! Let's add more translations 😊

**Pull request merged**